index.html

```diff
@@ -674,50 +773,49 @@ <h4>createDisclosureData</h4>
 (|proof|), an array of JSON pointers to use to selectively disclose
 statements (|selectivePointers|), an OPTIONAL BBS
 |presentationHeader| (byte array that defaults to an empty byte array if
-not present),
-an OPTIONAL
-|commitment_with_proof| (a byte array), an OPTIONAL |pid| value (a byte array),
+not present), a |featureOption| indicator, additional inputs as required by
+the |featureOption| (see <a href="#add-derived-proof-bbs-2023">Add Derived Proof</a>),
 and any custom JSON-LD API options
 (such as a document loader). A single object, <em>disclosure data</em>, is
-produced as output, which contains the `bbsProof`, `labelMap`,
-`mandatoryIndexes`, `selectiveIndexes`, `presentationHeader`, and
-`revealDocument` fields.
+produced as output, which contains the |bbsProof|, |labelMap|,
+|mandatoryIndexes|, |selectiveIndexes|, |presentationHeader|,
+|revealDocument|, and |pseudonym| (if computed) fields.
           </p>
 
           <ol class="algorithm">
             <li>
-Initialize `bbsSignature`, `bbsHeader`, `publicKey`, `hmacKey`,
-`mandatoryPointers`, and the optional feature parameters `pid` and
-`signer_blind` to the values of the associated properties in the object
+Initialize |bbsSignature|, |bbsHeader|, |publicKey|, |hmacKey|,
+|mandatoryPointers|, and the optional feature parameters |pid| and
+|signer_blind| to the values of the associated properties in the object
```

Comment:
Does it matter that `|pid|` and `|signer_blind|` are optional, here? Or should they just always be initialized as described?
```suggestion
Initialize |bbsSignature|, |bbsHeader|, |publicKey|, |hmacKey|,
|mandatoryPointers|, |pid|, and
|signer_blind| to the values of the associated properties in the object
```

Reply:
Hi @TallTed, in section "4.4 Optional Feature Summary" Table 2, I summarize that under different options the `pid` or (exclusive) the `signer_blind`, or neither would be used.  That is `paseBaseProof` returns different items under different options (this is called in step 1 of `createDisclosureData`. These are then used later in step 10 of  `createDisclosureData` where different processing is done based on the `featureOption` value.